security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultBuilder.java

```diff
@@ -280,28 +280,21 @@ public SecurityAnalysisResultBuilder endContingency() {
         }
     }
 
-    public class OperatorStrategyResultBuilder extends AbstractLimitViolationsResultBuilder<OperatorStrategyResultBuilder> {
+    public class OperatorStrategyResultBuilder {
 
         private final OperatorStrategy strategy;
 
-        private PostContingencyComputationStatus status = PostContingencyComputationStatus.CONVERGED;
+        private final List<OperatorStrategyResult.ConditionalActionsResult> conditionalActionsResult = Collections.synchronizedList(new ArrayList<>());
```

Comment:
Is it on purpose? Do we really support concurrent build of operator strategies?

Reply:
postContingencyResults and operatorStrategyResults both use synchronizedList so I figured there was some concurrent use somewhere and followed the same pattern. Maybe unecessary ?

Reply:
yes unnecessary